openmmtools/tests/test_integrators.py

```diff
@@ -206,6 +206,39 @@ def test_vvvr_shadow_work_accumulation():
     assert('shadow_work' not in names_of_globals)
 
 
+def test_vvvr_protocol_work_accumulation():
```

Comment:
I strongly recommend testing the accumulated work explicitly by comparing with `getState(getEnergy=True)`. I *thought* I had a test for this in `saltswap` here:

https://github.com/choderalab/saltswap/blob/master/saltswap/tests/test_integrators.py#L185-L206

But it's not exactly the same as testing the internally accumulated work. Anyway, something that tests for that would be good. (I'll certainly be adding to my `saltswap` tests.)